include/unifex/let_error.hpp

```diff
@@ -85,21 +85,26 @@ class _rcvr<Source, Func, Receiver>::type final {
     (requires receiver_of<Receiver, Values...>)
   void set_value(Values... values) noexcept(
       is_nothrow_receiver_of_v<Receiver, Values...>) {
-    UNIFEX_ASSERT(op_ != nullptr);
-    unifex::deactivate_union_member(op_->sourceOp_);
-    unifex::set_value(std::move(op_->receiver_), (Values&&)values...);
+    // local copy, b/c deactivate_union_member deletes this
+    auto op = op_;
+    UNIFEX_ASSERT(op != nullptr);
+    unifex::deactivate_union_member(op->sourceOp_);
+    unifex::set_value(std::move(op->receiver_), (Values&&)values...);
```

Comment:
This is a `move` and I _think_ we've settled on using `std::move` for moves.
```suggestion
    unifex::set_value(std::move(op->receiver_), std::move(values)...);
```